src/Pages/ShopPage.js

```diff
@@ -1,21 +1,20 @@
-import React from "react"
-import { Header } from "../Components/Header/Header"
-import { Aside } from "../Components/Aside/Aside"
-import { ContentContainer } from "../Components/ContentContainer/ContentContainer"
-import { MainContainer } from "../Components/MainContainer/MainContainer"
-import { Footer } from "../Components/Footer/Footer"
-export default function ShopPage()
-{
-    return(
-        <>
-        <Header/>
-        <main style ={{marginBottom: "50px"}}>
+import React from 'react';
+import { Header } from '../export';
+import { Aside } from '../export';
+import { ContentContainer } from '../export';
+import { MainContainer } from '../export';
+import { Footer } from '../export';
+export function ShopPage() {
+  return (
+    <>
+      <Header />
+      <main style={{ marginBottom: '50px' }}>
         <MainContainer>
-            <Aside/>
-            <ContentContainer/>
+          <Aside />
```

Comment:
Suggestion: you can put some of these components directly on app.js if they're being used on every page.